### DISCUSSION TITLE
Question on proper sizing for deployment

### DISCUSSION THREAD

**Discussion by wicknet** [Security-Onion-Solutions/securityonion#4277] [Unsupported Versions]
Kudos on releasing this superb IDS that just works out of the box, unlike other solutions out there that require extensive customization and integration. 

Have read the documentation but still confused about proper sizing. For a school network with ~800 devices for 1000 users, using a 1Gb sniffing port in promiscuous mode connected to a 10Gb span port on the core switch, should I go for one VM in standalone mode, or 1 master with 1 heavy node, or 1 master, 1 forward and 1 search node? 

Have tried the 1st 2 options already (standalone and master/heavy) and found no discernible differences in performance. Both VM's were running on ESXI v6.5 with 8 vCPU's, 64GB RAM and 1Tb RAID 10 drive storage allotted.

Any advice would be deeply appreciated. If the recommendation is for the ideal Master/Forward/Search deployment, please also suggest ideal vCPU, vRAM and storage resources to be allocated to each VM. Don't want to allot the max possible if it's not going to be used.

Thanks!

**Comment by Mastadamus** [comment]
1 tb will not provide a lot of space for pcaps. I have 25 devices being watched on a 1gb network and 1tb gets me about 3 weeks of pcaps.

**Reply by wicknet** [reply]
Thank you.. that's helpful. I can increase the HDD size easily. 

Assuming I need only 7 days of data retention, should I deploy 1 standalone or 3 separate VM's and what's their recommended hardware?
